lib/mysql.js

```diff
@@ -127,21 +121,16 @@ require('util').inherits(MySQL, SqlConnector);
  * @param {String} sql The SQL statement
  * @param {Function} [callback] The callback after the SQL statement is executed
  */
-MySQL.prototype.query = function (sql, callback) {
+MySQL.prototype.executeSQL = function (sql, params, options, callback) {
```

Comment:
`executeSQL` is the extension point for the SqlConnector to implement the `execute` API, which does some normalization. See https://github.com/strongloop/loopback-connector/blob/feature/connector-refactor/lib/sql.js#L363-L389